source/page/Main.tsx

```diff
@@ -0,0 +1,194 @@
+import { component, mixin, createCell, Fragment } from 'web-cell';
+import { observer } from 'mobx-web-cell';
+import { Jumbotron } from 'boot-cell/source/Content/Jumbotron';
+import { Button } from 'boot-cell/source/Form/Button';
+import { Image } from 'boot-cell/source/Media/Image';
+import { SpinnerBox } from 'boot-cell/source/Prompt/Spinner';
+import { Card } from 'boot-cell/source/Content/Card';
+import { Badge } from 'boot-cell/source/Reminder/Badge';
+
+import companies from '../data/members-china.json';
+import { member, proposal, Member, Proposal } from '../model';
+
+const thisYear = new Date().getFullYear();
+
+@observer
+@component({
+  tagName: 'main-page',
+  renderTarget: 'children'
+})
+export class MainPage extends mixin() {
+  connectedCallback() {
+    if (member.list.length < 1) member.getList();
+    if (proposal.finishedList.length < 1) proposal.getList();
+
+    super.connectedCallback();
+  }
+
+  renderMember = ({
+    avatar_url,
+    username,
+    url,
+    name,
+    company,
+    location,
+    bio
+  }: Member) => {
+    return (
+      <div className="col-12 col-sm-6 col-md-3 my-3">
+        <Card
+          image={avatar_url}
+          title={
+            <a
+              className="stretched-link"
+              title={username}
+              target="_blank"
+              href={url}
+            >
+              {name}
+            </a>
+          }
+        >
+          <dl>
+            <dt>组织</dt>
+            <dd>{company}</dd>
+            <dt>地区</dt>
+            <dd>{location}</dd>
+            <dt>自述</dt>
+            <dd>{bio}</dd>
+          </dl>
+        </Card>
+      </div>
+    );
+  };
+
+  renderProposal = ({
+    link,
+    name,
+    tags,
+    edition,
+    authors,
+    champions
+  }: Proposal) => {
+    const standard = tags[0].split('-')[1];
+
+    return (
+      <div className="col-12 col-sm-6 col-md-3 my-3">
+        <Card
+          className="h-100"
+          title={
+            link ? (
+              <a target="_blank" href={link}>
+                {name}
+              </a>
+            ) : (
+              name
+            )
+          }
+          header={
+            <div className="d-flex justify-content-around">
+              <Badge
+                color="primary"
+                target="_blank"
+                href={`https://www.ecma-international.org/publications/standards/Ecma-${standard}.htm`}
```

Comment:
```suggestion
                href={`https://tc39.es/ecma${standard}/`}
```

https://tc39.es/ecma262/
https://tc39.es/ecma402/